### DISCUSSION TITLE
Any way to query for hottest temperature across the globe today?

### DISCUSSION THREAD

**Discussion by HunterTTP** [open-meteo/open-meteo#909] [Q&A]
Is there a way to use your API to get the highest temperatures globally for the current day without querying each city individually? Any tips or suggestions would be great. Thanks!

**Comment by patrick-zippenfenig** [comment]
No, this is not feasible. It would require to each through the entire database which is fairly inefficient. Most of the time the hottest temperature will be Death Valley, somewhere in the middle of the Sarah desert or some other uninhabited place.
